js/repl/PluginConfig.js

```diff
@@ -57,6 +57,12 @@ const replDefaults: ReplState = {
   isPresetsTabExpanded: false,
   isSettingsTabExpanded: true,
   lineWrap: true,
+  statusBar: false,
+  status: {
+    compiled: {},
+    raw: {},
+  },
```

Comment:
Not sure how I feel about this nesting structure. Do we think we'll include things here _other_ than size? If not, do we _need_ to nest this 3 levels deep? (Couldn't we just do `rawSize` and `compiledSize`?)

Reply:
I think with new changes nested object won't be necessary :smile: